src/precision.hpp

```diff
@@ -16,12 +16,13 @@
 #pragma once
 
 #include <string>
-#include <unordered_map>
 
 #include <openvino/openvino.hpp>
 
 namespace ovms {
 
+using KFSDataType = std::string;
```

Comment:
Here you add alias, but in precision.cpp sometimes you use std::string and sometimes KFSDataType